app/src/main/java/com/example/nvdovin/weatherapp/ForecastActivity.java

```diff
@@ -20,22 +20,22 @@
     private RecyclerView recyclerView;
     int tempScale = 273;
     float separatorHeight = 5.5f;
```

Comment:
try to avoid using variable for layout dimention in java class, use  resources instead